recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools_scm
   run:
     - python >=3.9
-    - numpy
+    - numpy <2.0
```

Comment:
Usually this is bad practice, since the incompatibility is actually inherited, but in this case the module uses numpy a lot and it's not been possible to test any of it due to the other package incompatibilities. We can relax this once our dependencies become numpy 2.0 compatible and we are able to run our tests on numpy 2.0.